Dockerfile

```diff
@@ -14,12 +17,16 @@ RUN apk add --no-cache py-pip jq bash wget && pip install yq jsonschema
 COPY ./scripts/*.sh ./scripts/meta.yaml.schema /build/
 COPY /v3 /build/v3
 WORKDIR /build/
-RUN ./check_plugins_location.sh v3
-RUN ./set_plugin_dates.sh v3
-RUN ./check_plugins_viewer_mandatory_fields.sh v3
-RUN ./ensure_latest_exists.sh
-RUN ./index.sh v3 > /build/v3/plugins/index.json
-RUN chmod -R g+rwX /build
+
+# if only including the /latest/ plugins, apply this line to remove them from builder 
+# RUN rm -fr $(find /build/v3 -name 'meta.yaml' | grep -v "/latest/")
+
+RUN ./check_plugins_location.sh v3 && \
```

Comment:
I'd rather not combine the `RUN` command when in the builder image -- this is one of the issues multi-stage builds were designed to address. 

It's harder to read when a failure occurs (rather than `./ensure_latest_exists.sh` failed, we get the whole long string failed).

Reply:
well, true, but it also runs faster and creates fewer layers.

Reply:
Fair enough, it doesn't really matter to me.